great-idea/index.html

```diff
@@ -38,7 +39,7 @@ <h1>Innovation<br> On<br> Demand</h1>
     </section>
 
     <!-- Section Line -->
-    <div class = "line"> </div>
+    <div class = "line"> </div>    
```

Comment:
i would avoid using a div to create this effect. you could easily eliminate this "line" div and set a top-border on your "main-content" section. less code is always better.